config/packages/framework.yaml

```diff
@@ -18,3 +18,7 @@ framework:
     serializer:
         enabled: true
         enable_annotations: true
+
+    assets:
+        base_urls:
+            - 'https://kdti-statics.s3.amazonaws.com/'
```

Comment:
What do you think about replacing this URL with an environment variable?

Reply:
I don't think it's necessary. We can only set in the other environment folders like:
* `packages/local/framework.yml`
* `packages/dev/framework.yml`

Reply:
Actually, `local` should point to local directory. I suggest you to create an entry for that.

Reply:
I got it. Great!